packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -184,7 +185,11 @@ export const resolvers = {
     persistedPage: loggedInAndSignedUp(persistedPage),
     persistedPages: loggedInAndSignedUp(persistedPages),
     persistedBlocks: loggedInAndSignedUp(persistedBlocks),
-    persistedEntity: loggedInAndSignedUp(persistedEntity),
+    getPersistedEntity: loggedInAndSignedUp(getPersistedEntity),
+    getAllLatestPersistedEntities: loggedInAndSignedUp(
```

Comment:
We should be consistent about how we name query resolvers. Based on the GraphQL examples I've come across (for instance [in the apollo docs](https://www.apollographql.com/docs/apollo-server/data/resolvers/)), it seems to be standard to not use `get` as a prefix for query resolvers, so unless someone feels strongly to do it differently I'd suggest we stick with this approach for now.

(This also impacts the other query resolvers not introduced in this PR)

cc @CiaranMn @thehabbos007

Reply:
I would prefer that all queries and mutations had an `[action][resource]` format. it's completely unambiguous. 

in the type defs you can infer that a query will retrieve something, sure, but it is ambiguous elsewhere: does a function called `comment` retrieve a comment or create one? how about `archive`? by being explicit about what they do we can avoid this

Reply:
I'd align with Ciaran's proposal of `getX` as well. For what it's worth I named these methods for consistency with a lot of our _new_ resolvers, rather than any intentional decision about style.

I think this is probably worth a clean-up ticket because we are inconsistent across all the resolvers and we just need to improve consistency in general.

Reply:
Alright, I'm not sure whether we're missing some reason why GQL has standardised this resolver naming convention (possibly to distinguish between resolvers written like `presistedEntity { ... }` and `query getPersistedEntity { persistedEntity { ... } }`?), but I don't feel very strongly either way so happy to just go with `getPersistedEntity` for now.

Let's create that clean-up ticket and reference it in a todo then?

Reply:
Done https://github.com/hashintel/hash/pull/1242/commits/3b96c3dde4b8149d41f687ccad3ca03353265e6a